examples/iris.rb

```diff
@@ -0,0 +1,103 @@
+require 'numo/narray'
+require 'chainer'
+require "datasets"
```

Comment:
When run iris.rb, error occurs.
`require': cannot load such file -- datasets (LoadError)`

You need to add [red-datasets](https://github.com/red-data-tools/red-datasets) to gemspec